hack/preview.sh

```diff
@@ -86,3 +97,26 @@ git checkout $MY_GIT_BRANCH
 
 #set the local cluster to point to the current git repo and branch and update the path to development
 $ROOT/hack/util-update-app-of-apps.sh $MY_GIT_REPO_URL development $PREVIEW_BRANCH
+
+# Make sure we have a tekton-chains namespace
+echo "Checking to see if tekton-chains namespace exists"
+while ! kubectl get namespace tekton-chains &> /dev/null; do
+  echo -n .
+  sleep 3
+done
+
+echo "Setting chains to use cluster rekor server: https://rekor.$domain"
+kubectl patch configmap/chains-config -n tekton-chains --patch "{\"data\":{\"transparency.url\":\"https://rekor.$domain\"}}" --type=merge 
+# Delete the controller pod for chains to ensure that the configuration change gets picked up.
+echo "Restarting chains controller"
+kubectl delete pod -n tekton-chains -l app=tekton-chains-controller
+# If we have a rekor namespace, we should wait for the server to be available
+if kubectl get namespace &>/dev/null; then
+    kubectl delete pod -n enterprise-contract-service -l app.kubernetes.io/component=server
+    # # Wait to ensure that our rekor deployment is available by hitting the log API endpoint
+    # echo "Waiting for rekor server."
+    # while ! curl --fail --insecure --output /dev/null --silent "https://rekor.$domain/api/v1/log"; do
+    #     echo -n .
+    #     sleep 3
+    # done
```

Comment:
Is it intention to leave this in but commented out? Seems okay to do that, but just confirming. You could refine the comment to make it clear, e.g.:

    # Uncomment the following if you want to wait for rekor